src/esn.inbox.libs/app/services/inbox-restangular.service.spec.js

```diff
@@ -0,0 +1,26 @@
+/* global chai,sinon: false */
+const { module, inject } = angular.mock;
+const { expect } = chai;
+
+describe('The inboxRestangular module', function() {
+  let inboxRestangular;
+  let httpConfigurerMock;
+
+  beforeEach(function() {
+    httpConfigurerMock = {
+      setBaseUrl: () => {},
+      manageRestangular: sinon.stub()
+    };
+  });
+  beforeEach(module('esn.http'));
+  beforeEach(module('esn.inbox.libs'));
+  beforeEach(module(function($provide) {
+    $provide.value('httpConfigurer', httpConfigurerMock);
+  }));
+  beforeEach(inject(function(_inboxRestangular_) {
+    inboxRestangular = _inboxRestangular_;
+  }));
+  it('shoul register itself to httpConfigurer.managerRestangular', function() {
```

Comment:
Typo 'shoul register itself to httpConfigurer.managerRestangular' => 'should register itself to httpConfigurer.manageRestangular'

Reply:
fixed